test/parallel/test-eslint-prefer-assert-methods.js

```diff
@@ -7,31 +7,50 @@ const rule = require('../../tools/eslint-rules/prefer-assert-methods');
 
 new RuleTester().run('prefer-assert-methods', rule, {
   valid: [
-    'assert.strictEqual(foo, bar)',
-    'assert(foo === bar && baz)'
+    'assert.strictEqual(foo, bar);',
+    'assert.ok(foo === bar && baz);',
+    'assert.notStrictEqual(foo, bar);',
+    'assert.ok(foo !== bar && baz);',
+    'assert.equal(foo, bar);',
+    'assert.ok(foo == bar && baz);',
+    'assert.notEqual(foo, bar);',
+    'assert.ok(foo != bar && baz);'
   ],
   invalid: [
     {
-      code: 'assert(foo == bar)',
-      errors: [{ message: "'assert.equal' should be used instead of '=='" }]
+      code: 'assert(foo == bar);',
+      errors: [{
+        message: "'assert.equal' should be used instead of '=='"
+      }],
+      output: 'assert.equal(foo, bar);'
     },
     {
-      code: 'assert(foo === bar)',
+      code: 'assert(foo === bar);',
       errors: [{
         message: "'assert.strictEqual' should be used instead of '==='"
-      }]
+      }],
+      output: 'assert.strictEqual(foo, bar);'
     },
     {
-      code: 'assert(foo != bar)',
+      code: 'assert(foo != bar);',
       errors: [{
         message: "'assert.notEqual' should be used instead of '!='"
-      }]
+      }],
+      output: 'assert.notEqual(foo, bar);'
     },
     {
-      code: 'assert(foo !== bar)',
+      code: 'assert(foo !== bar);',
       errors: [{
         message: "'assert.notStrictEqual' should be used instead of '!=='"
-      }]
+      }],
+      output: 'assert.notStrictEqual(foo, bar);'
     },
+    {
+      code: 'assert(val);',
+      errors: [{
+        message: "'assert.ok' should be used instead of alias 'assert'"
+      }],
+      output: 'assert.ok(val);'
+    }
```

Comment:
I would not prefer `assert.ok` over `assert`. Both should be fine out of my perspective.

Reply:
I'm a bit confused here. What did you mean when you said extend for 'assert.ok()'?

Reply:
In general `assert()` should be treated identical to `assert.ok()`. And I meant the tests should be extended to test for both. Before I commented there were only tests for `assert()`.

Reply:
Thanks for clarifying this. I'll revert the added invalid test for `assert(val)`.

Reply:
Done.